### ISSUE TITLE
Incorrect results glitch

### ISSUE THREAD

**Issue by 123CK** [megadose/holehe#96]
I've found a glitch. It concerns Nike.com in connection with a Gmail. 

To reproduce: create a Nike account with a Gmail, run the Gmail on Holehe (it will correctly show a Nike account), then go to your Nike account settings and change the email on there into something else (other than Gmail provider), then run the Gmail again, and Holehe will still show (this time incorrectly) a Nike account. Same result even for randomized Gmail catchalls such as myname+1234@gmail

Nike must have changed something on their backend because this is definitely a new glitch. Bottom line is, whatever Nike did makes Holehe spit out incorrect info

**Comment by LucBerge** [comment]
Has the new email been verified ? It often takes some time (few days) to update a user database and sync the website with the new database.

If you use the old gmail address today, does holehe show a Nike.com account ?

**Comment by 123CK** [comment]
The Gmails in question have been removed (changed into something else in the respective accounts) from Nike several months ago. They subsequently didn't trigger on Holehe afterwards. Also those changes synced instantly. Now they showed up again. So we are for one way past a few days here a
![holehe](https://user-images.githubusercontent.com/108565116/178459545-90ddb86d-a713-433b-89fa-6f1ed8e9e7bc.jpg)
nd for two, something that was already gone basically reappeared even though its a false positive. What I also noticed: if the original Gmail was lets say jane.doe@, then every possible catchall will show a Nike acc on Holehe, even though there never has been an account associated with that specific catchall. If you reverse the situation, I'll use my own example in this case: my Nike Gmail was jane.doe+123@ before I swapped it into a whole another provider. Now my main jane.doe@gmail returns a Nike account on Holehe. I attached a screenshot. 

> Has the new email been verified ? It often takes some time (few days) to update a user database and sync the website with the new database.
> 
> If you use the old gmail address today, does holehe show a Nike.com account ?

The Gmails in question have been removed (changed into something else in the respective accounts) from Nike several months ago. They subsequently didn't trigger on Holehe afterwards. Also those changes synced instantly. Now they showed up again. So we are for one way past a few days here and for two, something that was already gone basically reappeared even though its a false positive. What I also noticed: if the original Gmail was lets say jane.doe@, then every possible catchall will show a Nike acc on Holehe, even though there never has been an account associated with that specific catchall. If you reverse the situation, I'll use my own example in this case: my Nike Gmail was jane.doe+123@ before I swapped it into a whole another provider. Now my main jane.doe@gmail returns a Nike account on Holehe.
